### DISCUSSION TITLE
What to include in Version control?

### DISCUSSION THREAD

**Discussion by robzed** [Sloeber/arduino-eclipse-plugin#1307] [Q&A] [answered]
I'm about to book the Sloeber project files into git... So, .project, .cproject, .sproject all look ok. Exclude /Release/ /sloeber.ino.cpp .settings/

What about spec.d ?

**Comment by jantje** [comment, marked as answer]
in 4.4 you should only check in the sloeber.cfg
When checking out check out as a standard project and then convert it to a sloeber project

**Reply by jantje** [reply]
so
do not check in .project, 
do not check in .cproject, 
do not check in .sproject 
do not check in  /Release/ 
do not check in  /sloeber.ino.cpp 
do not check in .settings/
do not check in  spec.d 
Check in sloeber.cfg (make it by going to project properties->arduino->other and select "share in version control" for all config you want to sshare

**Comment by jantje** [comment]
Here a video I just made https://www.youtube.com/watch?v=bqdw1L_8Gjs
